File: Learning/Java Basic/#6/Problem_58.java
//https://codelearn.io/learning/java-co-ban/1195637

public class Loop {
	public static void main(String[] args) {
	    int i = 1;
	    do {
	        if (i % 10 == 0)
	    		System.out.print(i + " ");
			i += 1;
	    } while (i <= 1000);
	}
}

File: Learning/Python3 Basic/#11/Problem_58.py
#https://codelearn.io/learning/python-co-ban/132319

n = int(input())

def sumOfAll(n):
    result = 0;
    for i in range(1, n):
        if (n % i == 0):
            result += i
    return result

print(sumOfAll(n))